apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/features/ObjectApiLiveTest.java

```diff
@@ -141,20 +142,17 @@ public void testCopyObject() throws Exception {
          // test exception thrown on bad source name
          try {
             destApi.copy(destinationObject, badSource, sourceObjectName);
-            fail("Expected CopyObjectException");
```

Comment:
Can you use try .. catch instead of `expectedExceptions`?  This narrows the exception handling scope:

http://konigsberg.blogspot.com/2007/11/testng-and-expectedexceptions-ive.html

(also the for loop above will not complete with this change)